unit/size_left_test.c

```diff
@@ -142,11 +142,10 @@ rx_size_left_err(void)
 	}
 
 	/* ep starts in a non-enabled state, may fail, should not SEGV */
-	fi_rx_size_left(ep);
+	(void) fi_rx_size_left(ep);
```

Comment:
Cast to void to make it obvious that we are purposely ignoring the return value.  In reading the man page again, perhaps this should return -FI_EOPBADSTATE?